lib/expenses/index.js

```diff
@@ -59,3 +59,12 @@ export const isSupportedExpenseType = (account, expenseType) => {
   const supportedTypes = getSupportedExpenseTypes(account);
   return supportedTypes.includes(expenseType);
 };
+
+export const getFilesFromExpense = expense => {
+  if (!expense) {
+    return [];
+  }
+
+  const items = expense.items?.filter(({ url }) => Boolean(url)) || [];
+  return [...items, ...expense.attachedFiles];
```

Comment:
On the collective expenses page:

![image](https://github.com/opencollective/opencollective-frontend/assets/1556356/cbfc6eca-59c9-47d0-9718-a0c70069a127)

`attachedFiles` can be null if you don't have permissions, so:

```suggestion
  return [...items, ...(expense.attachedFiles || [])];
```